test/ssl-tests/17-renegotiate.cnf.in

```diff
@@ -261,6 +261,37 @@ our @tests_tls1_2 = (
             "ResumptionExpected" => "No",
             "ExpectedResult" => "ServerFail"
         }
+    },
+    {
+        name => "allow-client-renegotiation",
+        server => {
+            "MaxProtocol" => "TLSv1.2",
+            "Options" => "ClientRenegotiation",
```

Comment:
You previously changed the test to turn this option on by default:

https://github.com/openssl/openssl/blob/eb1b66f00ca4e1fb6f9e815e8686768b6d81722d/test/ssl_test.c#L454-L481

Reply:
>you previous changed the test to turn this option on by default.

Schizophrenia? Senility?  Aargh.